vue-app/src/components/Cart.vue

```diff
@@ -409,6 +410,18 @@ export default class Cart extends Vue {
     return this.$web3.chainId
   }
 
+  get supportedChainId(): number {
+    return Number(process.env.VUE_APP_ETHEREUM_API_CHAINID)
+  }
+
+  get networkName(): string {
+    return CHAIN_INFO[this.supportedChainId].label
+  }
+
+  isCorrectNetwork(): boolean {
+    return this.supportedChainId === this.walletChainId
+  }
```

Comment:
I simplify a bit this logic. There was no need to check into the "default" `jsonRpcNetwork`. We can check the supported chainId by reading the env var `VUE_APP_ETHEREUM_API_CHAINID`.